src/main/java/org/spongepowered/common/mixin/api/mcp/entity/passive/VillagerTrades_ItemsForEmeraldsTrade_API.java

```diff
@@ -43,33 +42,36 @@
 // added as the only thing needing to be done is a simple default implementation
 // with an empty MerchantRecipeList and diff the list with an empty one and
 // provide the resulting diff'ed MerchantRecipe (TradeOffer) as the result.
-@Mixin(VillagerEntity.ListItemForEmeralds.class)
-public class EntityVillager_ListItemForEmeraldsMixin_API implements TradeOfferGenerator {
-
-    @Shadow public ItemStack itemToBuy;
-    @Shadow public VillagerEntity.PriceInfo priceInfo;
+//
+// i509VCB: Yes the classes which implement ITrade, like ItemsForEmeraldsAndItemsTrade are package private
+// For clarification this trade is Emeralds -> Item(s) such as a generic Emeralds -> Bread trade.\
+// TODO: These need a new home
+@Mixin(targets = "net/minecraft/entity/merchant/villager/VillagerTrades$ItemsForEmeraldsTrade")
+public class VillagerTrades_ItemsForEmeraldsTrade_API implements TradeOfferGenerator {
+    @Shadow public ItemStack field_221208_a; // Item being sold to the customer
```

Comment:
Don't leave in comments, they just pollute. Instead, name them via the MCP bot with an IRC client. Ask @gabizou how.